### PR TITLE
Give advice on managing GitHub notifications in CONTRIBUTING.

### PR DESCRIPTION
**Pull request by Zimmi48**
**Kind:** documentation

This advice should be useful for core developers who still have trouble with their notifications, sub-system maintainers and occasional contributors alike.

### CODE REVIEW

**Comment by Zimmi48** [comment]
@SkySkimmer The fact that GitHub didn't ask any code owner review is because I'm the primary owner for that file. And the reason I assigned @maximedenes is because he is the secondary owner.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -54,6 +54,20 @@ The sources for the [Coq reference manual](https://coq.inria.fr/distrib/current/
 
 You may also contribute to the informal documentation available in [Cocorico](https://github.com/coq/coq/wiki) (the Coq wiki), and the [Coq FAQ](https://github.com/coq/coq/wiki/The-Coq-FAQ). Both of these are editable by anyone with a GitHub account.
 
+## Following the development
+
+If you want to follow the development activity around Coq, you are encouraged
+to subscribe to the [Coqdev mailing list](https://sympa.inria.fr/sympa/info/coqdev).
+The activity on this mailing list is quite reasonable.
```
Comment (maximedenes) [inline CONTRIBUTING.md]: I'm not sure how explicit "quite reasonable" is. Maybe "has reasonably low traffic" or something like that instead?

**Comment by Zimmi48** [comment]
This was updated with your suggestion.

**Review by maximedenes — APPROVED** [review]
LGTM, thanks!

**Pull request merged**